pkgs/by-name/gi/gitea/package.nix

```diff
@@ -88,7 +88,7 @@ in buildGoModule rec {
   '';
 
   passthru = {
-    data-compressed = lib.warn "gitea.passthru.data-compressed is deprecated. Use \"compressDrvWeb gitea.data\"." (compressDrvWeb gitea.data);
+    data-compressed = lib.warn "gitea.passthru.data-compressed is deprecated. Use \"compressDrvWeb gitea.data\"." (compressDrvWeb gitea.data { });
```

Comment:
When I run this on master, I get a bunch of `Permission denied` errors. This commit doesn't change that outcome.

Reply:
I'll have a look at this separately.

Reply:
That's because of `lndir`. Now I realized why I didn't use it. I will update `compressDrv` soon to not use `lndir`.

The culprit is here: `-compressed` is the derivation being built. `public` should be a directory (so `compressDrv` can write files there), but is instead a symlink:

```
$ ls -lh /nix/store/yvsl8qm4y4c0xhfjq530ngnw1b54wj85-gitea-1.22.1-compressed/public
lrwxrwxrwx 1 nixbld1 nixbld 72 Aug 18 22:28 /nix/store/yvsl8qm4y4c0xhfjq530ngnw1b54wj85-gitea-1.22.1-compressed/public -> /nix/store/6yiik4x5r8midqp8shvc3pfvpac9bj6a-gitea-frontend-1.22.1/public
```

Reply:
#335670